.github/workflows/label-commenter.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Automatically respond to any issues/pull requests that have the given labels assigned.
+#
+name: Label Commenter
+
+on:
+  issues:
+    types:
+      - labeled
+      - unlabeled
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  comment:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Label Commenter
+        uses: peaceiris/actions-label-commenter@v1
```

Comment:
This is an action from the [marketplace](https://github.com/marketplace/actions/label-commenter), I need to confirm if the actions are also security reviewed by Github, or if it's only apps that just reviewed:

https://developer.github.com/marketplace/getting-started/requirements-for-listing-an-app-on-github-marketplace/#security